src/Field/Field.tsx

```diff
@@ -39,11 +39,17 @@ export const Field = ({
       {label && (
         <Box mb={{ custom: outlined ? 4 : 0 }}>
           {renderAsTitle ? (
-            <Title htmlFor={id}>{label}</Title>
+            <Text tag="label" typo="heading-small" htmlFor={id} mb="16px">
+              {label}
+            </Text>
           ) : (
-            <Text tag="label" color="subtext" typo="label" htmlFor={id}>
+            <Text tag="label" typo="label" color="subtext" htmlFor={id}>
               {label}
-              {required && <Asterisk>*</Asterisk>}
+              {required && (
+                <Text tag="span" typo="body-small" color="error">
+                  *
+                </Text>
+              )}
```

Comment:
This isn't needed for the title version?

Reply:
I haven't touched anything is this component. It was like this even before and I don't know how it works from a design point of view since I never saw an example with title with required